editorSupport/atom-reason/src/AtomReasonOutline.re

```diff
@@ -26,8 +29,7 @@ let makeTokens data => {
     | "Class"
     | "Constructor"
     | "Exn"
-    | "Module"
-    | "Signature" => buildToken "class-name" name
```

Comment:
Yeah that's weird. It's described [here](https://github.com/the-lambda-church/merlin/blob/master/doc/dev/PROTOCOL.md#completion)